consensus/obcpbft/pbft-core.go

```diff
@@ -252,6 +255,11 @@ func newPbftCore(id uint64, config *viper.Viper, consumer innerStack) *pbftCore
 	} else {
 		logger.Info("PBFT null requests disabled")
 	}
+	if instance.viewChangePeriod > 0 {
```

Comment:
I don't think this is a strong enough check. What happens if the sequence number is set to below the checkpoint interval?  In this case, my rough analysis of the code shows that the network would just constantly and forever change view, without ever reaching a new checkpoint.  If we want to go with the 'every n sequence numbers' approach, rather than 'every n checkpoints', I think we need to have better test coverage, making sure we include intervals that are above and below the checkpoint size as well.